internal/cephfs/util.go

```diff
@@ -67,8 +67,8 @@ func (cs *ControllerServer) validateCreateVolumeRequest(req *csi.CreateVolumeReq
 		return status.Error(codes.InvalidArgument, "volume Capabilities cannot be empty")
 	}
 
-	for _, cap := range reqCaps {
```

Comment:
instead of full name, I would like to go with `c`

Reply:
Why? with `capability` its explicit and clear what we are ranging on.

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Reply:
`More unusual things and global variables need more descriptive names` -> this is the case here, Isnt it? I think thats why pick `cap` in first place. 

@nixpanic whats your thought here? 
My preference goes with `capability` though to make it clear.

Reply:
`c` is fine, `req.VolumeCapabilities` is very explicit, and `ReqCaps` does not have the full name either, assuming the reader already knows what `Caps` refers to.

But, `capability` isn't an issue for me.

Reply:
Considering we do have assignments like https://github.com/ceph/ceph-csi/pull/1600/files#diff-9eb942b29a818285935a4ffb1f6584a4d9e86317fbaf07a40a5029b35c2713b7R77 below , keeping it on `capability`. If really needed I can change to `c`.